test/.eslintrc.yml

```diff
@@ -1,8 +1,4 @@
-extends: airbnb
-plugins:
-  - react
 env:
-  node: true
```

Comment:
`env` setting is for letting eslint ignore global variables of each environment. `node` is to ignore `module`, `process`, etc.

The `airbnb/base` in outer .eslintrc has already done that for us (https://github.com/airbnb/javascript/blob/master/packages/eslint-config-airbnb-base/rules/node.js).